src/main/java/com/xpanxion/java/assignments/student10/Worker10.java

```diff
@@ -0,0 +1,22 @@
+package com.xpanxion.java.assignments.student10;
+
+import com.xpanxion.java.assignments.DataAccess;
+import java.util.stream.Collectors;
+
+public class Worker10 {
+
+    public void ex1() {
+        System.out.println("Ex. 1...");
+        var productList = DataAccess.getProducts();
+        var departmentList = DataAccess.getDepartments();
+
+        var newProductList = productList.stream().map(product -> {
+            var departmentId = product.getDepartmentId();
+            var departmentName = departmentList.get(departmentId).getName();
```

Comment:
This only works if the department Ids are in asc order, right?  What if they are not in order?

Reply:
ahh that makes sense. k I'll figure out another way

Reply:
ok I think I got it this time